diff -p -up libprelude-1.0.0/prelude-admin/prelude-admin.c~ ./libprelude-1.0.0/prelude-admin/prelude-admin.c
--- libprelude-1.0.0/prelude-admin/prelude-admin.c~	2010-01-29 11:01:15.000000000 +0000
+++ libprelude-1.0.0/prelude-admin/prelude-admin.c	2012-05-31 23:06:53.000000000 +0000
@@ -57,7 +57,6 @@
 #include <errno.h>
 #include <gnutls/gnutls.h>
 #include <gnutls/x509.h>
-#include <gnutls/extra.h>
 
 #include "common.h"
 #include "config-engine.h"
diff -p -up libprelude-1.0.0/prelude-admin/server.c~ ./libprelude-1.0.0/prelude-admin/server.c
--- libprelude-1.0.0/prelude-admin/server.c~	2010-01-29 11:01:15.000000000 +0000
+++ libprelude-1.0.0/prelude-admin/server.c	2012-05-31 23:07:11.000000000 +0000
@@ -39,7 +39,6 @@
 
 #include <gcrypt.h>
 #include <gnutls/gnutls.h>
-#include <gnutls/extra.h>
 
 #include "prelude-client.h"
 #include "prelude-error.h"
